evalml/pipelines/components/estimators/regressors/prophet_regressor.py

```diff
@@ -0,0 +1,155 @@
+import copy
+
+import numpy as np
+import pandas as pd
+from skopt.space import Real
+
+from evalml.model_family import ModelFamily
+from evalml.pipelines.components.estimators import Estimator
+from evalml.problem_types import ProblemTypes
+from evalml.utils import import_or_raise, infer_feature_types
+from evalml.utils.gen_utils import classproperty, suppress_stdout_stderr
+
+
+class ProphetRegressor(Estimator):
+    """
+    Prophet is a procedure for forecasting time series data based on an additive model where non-linear trends are fit with yearly, weekly, and daily seasonality, plus holiday effects.
+    It works best with time series that have strong seasonal effects and several seasons of historical data. Prophet is robust to missing data and shifts in the trend, and typically handles outliers well.
+
+    More information here: https://facebook.github.io/prophet/
+
+    """
+
+    name = "Prophet Regressor"
+    hyperparameter_ranges = {
+        "changepoint_prior_scale": Real(0.001, 0.5),
+        "seasonality_prior_scale": Real(0.01, 10),
+        "holidays_prior_scale": Real(0.01, 10),
+        "seasonality_mode": ["additive", "multiplicative"],
+    }
+    model_family = ModelFamily.PROPHET
+    supported_problem_types = [ProblemTypes.TIME_SERIES_REGRESSION]
+
+    def __init__(
+        self,
+        date_column="ds",
+        changepoint_prior_scale=0.05,
+        seasonality_prior_scale=10,
+        holidays_prior_scale=10,
+        seasonality_mode="additive",
+        random_seed=0,
+        stan_backend="CMDSTANPY",
```

Comment:
This isn't meant to be changed right? I thought the plan was to only support prophet with the CMDSTANPY backend?

Reply:
So we need this to be specified so we can unit test it. Unfortunately the parameter doesn't get included in the attributes list unless it's passed into the Prophet component